cloudbridge/providers/azure/services.py

```diff
@@ -544,7 +521,7 @@ def get(self, bucket_id):
     def list(self, limit=None, marker=None):
         buckets = [AzureBucket(self.provider, bucket)
                    for bucket
-                   in self.provider.azure_client.list_containers()[0]]
```

Comment:
The API is now returning a [ItemPaged object](https://docs.microsoft.com/en-us/python/api/azure-core/azure.core.paging.itempaged?view=azure-python)